code/game/machinery/magnet.dm

```diff
@@ -21,7 +21,7 @@
 	var/code = 0 // frequency code, they should be different unless you have a group of magnets working together or something
 	var/turf/center // the center of magnetic attraction
 	var/on = FALSE
-	var/pulling = 0
+	var/pulling1 = 0

```

Comment:
Rename to something more readable, like magnetpulling.

Reply:
Also 0 and 1 to FALSE/TRUE.

Reply:
mkay.I'll fix it, when there will be several more reviews to less clutter commit history.

Reply:
I commented on this before in the other pr, why not keep the name you changed it to there?